config/env_sample.json

```diff
@@ -168,22 +168,20 @@
         "videos": {"types": ["leccap"], "icon": "fas fa-video fa-lg"}
     },
     "/* Institute specific urls add or remove to your needs, will be used for Resources view": "*/",
-    "RESOURCE_URLS": {
-        "canvas": {"prefix": "https://example.instructure.com/files/", "postfix": "/download?download_frd=1"},
-        "leccap": {"prefix": "https://leccap.engin.umich.edu/leccap/viewer/r/", "postfix": ""}
-    },
     "/*cron queries to pull the data from various caliper data sources replace if not relevant with your institution": "*/",
     "RESOURCE_ACCESS_CONFIG": {
         "canvas_data": {
             "query": "SELECT 'canvas' AS resource_type, CAST(SUBSTR(JSON_EXTRACT_SCALAR(event, '$.object.id'), 35) AS STRING) AS resource_id, CAST(SUBSTR(JSON_EXTRACT_SCALAR(event, '$.membership.member.id'), 29) AS INT64) AS user_id, CAST(SUBSTR(JSON_EXTRACT_SCALAR(event, '$.group.id'),31) AS INT64) AS course_id, COALESCE(JSON_EXTRACT_SCALAR(event, '$.object.extensions[\\'com.instructure.canvas\\'].asset_name'), JSON_EXTRACT_SCALAR(event, '$.object.extensions[\\'com.instructure.canvas\\'].filename'), JSON_EXTRACT_SCALAR(event, '$.object.name'), 'attachment') as name, datetime(EVENT_TIME) as access_time FROM event_store.events where JSON_EXTRACT_SCALAR(event, '$.edApp.id') IN UNNEST(['http://m.canvas.umich.edu/' , 'http://umich.instructure.com/' ]) and type = 'NavigationEvent' and SUBSTR(JSON_EXTRACT_SCALAR(event, '$.object.id'),24,10) = 'attachment' and JSON_EXTRACT_SCALAR(event, '$.action') = 'NavigatedTo' and JSON_EXTRACT_SCALAR(event, '$.membership.member.id') is not null and SUBSTR(JSON_EXTRACT_SCALAR(event, '$.group.id'),31) IN UNNEST(@course_ids)",
             "app_display_name": "Canvas Files",
-            "canvas_course_id_format": "LONG"
+            "canvas_course_id_format": "LONG",
+            "urls:": {"prefix": "https://example.instructure.com/files/", "postfix": "/download?download_frd=1"}
```

Comment:
There is a typo here in the `"urls:"` key.

Reply:
```suggestion
            "urls": {"prefix": "https://example.instructure.com/files/", "postfix": "/download?download_frd=1"}
```